### ISSUE TITLE
What do you think about Google interview Today

### ISSUE THREAD

**Issue by JianweiWangs** [mxcl/AmA#14]

**Comment by mxcl** [comment]
I have always thought differently than is commonly imagined. Which I understand, the tweet seemed flippant and outraged. But the thing is, tweets always seem like that. Recall at the time that I had 3,000 followers and the majority of things I said went unnoticed. I had just gotten off the phone with Google and found it a little ridiculous so I typed out a quick rant and sent. I didn’t check back for an hour.

Rewind some more, I was in-between jobs and not sure what I wanted to do with myself. I asked my wife for advice and she said “why don’t you reply to one of those Google recruiter emails you always get?” And so I did. I had no grand desire to work there, in fact as a do-it-myself, open source-y type, the idea of working for a megacorp was not very appealing at all.

I asked the recruiter if they were going to ask me any computer science since ”I don’t know most of it, I have a Chemistry degree” and he said something along the lines of “no, not much anyway”. So I assumed I would be asked on topics I had expertise in.

Straight up, the first interview was the famous one.

So I was not prepared, and really that was my own fault. But I never was that bothered about whatever the outcome would be, despite appearances in the tweet. I would have enjoyed working there, and I think I would have done well there. But really I prefer startups or being in control of my own agency.

I equally regret and don’t regret the tweet. I don’t want to be famous for a tweet, I want to be known for my work. I feel bad that I stood up and was one of those obnoxious people who point fingers, get the mob behind them and make someone look bad in a way they cannot counter. However I feel it also encouraged a lot of good conversation about interviews in this industry, and for many talented engineers that have taught themselves and only learned computer science when they needed it (like myself) that is a good thing.

**Comment by MarcSteven** [comment]
@mxcl  Yeah it's a wonderful process and experience for more developers.Yeah sometimes we should try and then prove something...Actions can lead you to move forward ..
